packages/@sanity/cli/src/util/clientWrapper.js

```diff
@@ -1,5 +1,7 @@
-import client from '@sanity/client'
 import getUserConfig from './getUserConfig'
+import client from '@sanity/client'
+
+const generateHelpUrl = require('@sanity/generate-help-url')
```

Comment:
Probably want to use import to stay consistent with the ones above?

Reply:
oops! fixed!